recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -204,6 +204,12 @@ do_deploy() {
         echo "dtoverlay=dwc2,dr_mode=peripheral" >> $CONFIG
     fi
 
+    # DWC2 USB host mode support
```

Comment:
Looks good. Can you please also update the documentation?

Reply:
done